openapi/components/schemas/LimitedUser.yaml

```diff
@@ -51,6 +51,9 @@ properties:
     type: string
   friendKey:
     type: string
+  last_login:
+    type: string
+    format: date-time
```

Comment:
`last_login` is not present in the response of:
https://vrchat.com/api/1/users?search=tupper

Reply:
Upon further investigation, I discovered and successfully tested that the field is only accessible if the authenticated account is friends with the target account.